elide-integration-tests/src/test/java/example/Generate.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package example;
+
+import com.yahoo.elide.annotation.Include;
+
+import org.hibernate.annotations.Generated;
+import org.hibernate.annotations.GenerationTime;
+
+import lombok.Setter;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+@Entity
+@Include(rootLevel = true)
+public class Generate {
```

Comment:
Should we move this to elide-example-models?  Or do you think there is a case for keeping these kinds of tests in integration-tests?

Reply:
`@Generated` annotation is not available to ` elide-example-models`